################################################################################
 main_driver_500

 Hash: main_driver_500_d9fb06f13a39cd0e397b9d7266f7021f0baf4404

 To attempt to reproduce this bug using restler, run restler with the command
 line option of --replay_log <path_to_this_log>.
 If an authentication token is required, you must also specify the token_refresh_cmd.

 This log may contain specific values for IDs or names that were generated
 during fuzzing, using the fuzzing dictionary. Such names will be re-played
 without modification. You must update the replay log manually with any changes
 required to execute the requests in your environment (for example, replacing
 pre-created account, subscription, or other resource IDs, as needed).
################################################################################

-> GET //miscare/statistics?time_period=1.23 HTTP/1.1\r\nAccept: application/json\r\nHost: localhost:42178\r\n\r\n
! producer_timing_delay 0
! max_async_wait_time 20
PREVIOUS RESPONSE: 'HTTP/1.0 500 INTERNAL SERVER ERROR\r\nContent-Type: text/html; charset=utf-8\r\nContent-Length: 290\r\nServer: Werkzeug/2.0.2 Python/3.7.9\r\nDate: Thu, 03 Feb 2022 00:37:51 GMT\r\n\r\n<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'

